src/vs/workbench/contrib/webview/browser/webviewMessages.d.ts

```diff
@@ -17,6 +17,10 @@ type KeyEvent = {
 	repeat: boolean;
 }
 
+type WebViewDragEvent = {
```

Comment:
I tried to type this `DragEvent` instead of `WebViewDragEvent` but, the tests failed. I can only assume this is due to a collision with the standard DOM `DragEvent` type. I would have expected a typescript error though, this was hard to figure out.